docs/reference/advanced_configuration.rst

```diff
@@ -306,19 +306,19 @@ If you want to use the Tab Menu in a different way, you can replace the Menu Tem
 Translations
 ^^^^^^^^^^^^
 
-The translation parameters and domain can be customised by using the
-``translation_domain`` and ``translation_parameters`` keys of the extra array
+The label translation parameters and domain can be customised by using the
+``label_translation_parameters`` and ``label_catalogue`` keys of the extra array
 of data associated with the item, respectively::
 
     $menuItem->setExtras([
-        'translation_parameters' => ['myparam' => 'myvalue'],
-        'translation_domain' => 'My domain',
+        'label_translation_parameters' => ['myparam' => 'myvalue'],
```

Comment:
Why did you change this to `label_catalogue`? Is there a reason why we could not keep `translation_domain` or why the new parameter is not called `label_catalogue_parameters` ?

Reply:
If you look at the code, you can see
```
{%- set translation_domain = item.extra('label_catalogue', 'messages') -%}
```
So `translation_domain` was not working, he just fixed the doc.

IMHO catalogue/label_catalogue/default_catalogue should be deprecated in favor of Symfony-like options. I already created an issue for this https://github.com/sonata-project/SonataAdminBundle/issues/6116

Reply:
The changelog should help to understand what is fixed or improved, but it is missing in this PR.
Could you please solve that @pavol-tk? The instructions are available at [CONTRIBUTING.md](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/CONTRIBUTING.md#the-changelog).